Gruntfile.js

```diff
@@ -575,6 +584,29 @@ module.exports = function (grunt) {
         ]);
     });
 
+    grunt.registerMultiTask('swPrecache', function() {
+        var done = this.async();
+        var rootDir = this.data.rootDir;
+        var files =  (function() {
```

Comment:
Yes, you could do this, if we were smarter about how we copied things to `dist/`.  But we aren't smart.  Everything that you tell the SW to precache, it will download and cache.  So if you ask for more than you need, you take a big hit on network installation on first run.  If you look at what's in `dist/`, we copy all kinds of crap for CodeMirror and other things.  We could fix that, but it's going to basically involve doing what I'm doing here: keeping a static list of things we care about.  So I'm not sure it's worth fixing.  There's no harm copying them to `dist/`; but we don't really want things getting downloaded/cached on the browser that we know we'll never use.